libobs-opengl/gl-egl-common.c

```diff
@@ -215,6 +215,114 @@ gl_egl_create_dmabuf_image(EGLDisplay egl_display, unsigned int width,
 	return texture;
 }
 
+EGLBoolean query_dmabuf_implicit(EGLDisplay egl_display)
+{
+	UNUSED_PARAMETER(egl_display);
+	return EGL_EXT_image_dma_buf_import > 0;
```

Comment:
I don't understand this check. What is `EGL_EXT_image_dma_buf_import` supposed to be? A function pointer? The version of the extension?

Reply:
It's a macro to signal, that an extension is present (1 or 0). Funny thing: since we use glad, this will always be true, so worst thing is, the gpu doesn't know  anything about modifiers, import fails and we can test if the fallback to SHM works. This check is for correctness reasons, which should be changed if glad gets replaced, or we have to do some direct EGL calls if testing shows it's necessary.